lib/osv/repos.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Google LLC
```

Comment:
nit: Maybe repo_manger is a slightly better name ?

Reply:
This is more consistent with other existing modules, e.g. "sources.py" instead of "source_manager.py"